From 70703cb2f1059adbc740b243b4003c9f3c90dbe3 Mon Sep 17 00:00:00 2001
From: "Ranjan, Rajani" <rajani.ranjan@intel.com>
Date: Wed, 26 Apr 2023 13:58:17 +0530
Subject: [PATCH] mixin changes to support sensors aidl hal .

sensors package is customized from
android.hardware.sensors@2.0-service.intel
to android.hardware.sensors@aidl-service.intel

Tracked-On: OAM-110600
Signed-off-by: Ranjan, Rajani <rajani.ranjan@intel.com>
---
 groups/sensors/mediation/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/sensors/mediation/product.mk b/groups/sensors/mediation/product.mk
index 3441167..3c8e3ab 100644
--- a/groups/sensors/mediation/product.mk
+++ b/groups/sensors/mediation/product.mk
@@ -4,7 +4,7 @@ endif
 
 # Sensors HAL modules
 PRODUCT_PACKAGES += \
-	android.hardware.sensors@2.0-service.intel
+	android.hardware.sensors@aidl-service.intel
 
 {{#enable_sensor_list}}
 PRODUCT_COPY_FILES += \
-- 
2.41.0

